index.bs

```diff
@@ -1082,6 +1082,28 @@ spec: webidl
       permission is the permission associated with the usage of the
       asynchronous methods in the [[clipboard-apis]].
     </p>
+    <dl>
+      <dt>
+        <a>permission descriptor type</a>
+      </dt>
+      <dd>
+        <pre class='idl'>
+          dictionary ClipboardPermissionDescriptor : PermissionDescriptor {
+            boolean fullAccess = false;
```

Comment:
Are you sure that in the future you won't have a requirement for more granular access? Did you consider instead:

```JS
{
   access: "full",
}
```

Reply:
We don't believe that we'll need to be more granular than write (basic) and read/write (full), partially because that would be hard to surface to the user in a clear way.

Having said that, if you think it would be better to have this be `{ access: "write" }` and `{ access: "full" }` (or "read-write") then I don't have a problem with that.  The default `{ clipboard }` would be equivalent to `{ name: "clipboard", access: "write" }` and `full` would be stronger than `write`.

In the future, we may want to add another dimension (e.g., to control access to particular mimetypes), but that would be in a separate field.

Reply:
Thanks for the clarification - that makes sense. I don't have a strong opinion, so I think it's probably ok to just have `fullAccess` to stand for "read-write" (and missing for just write). 

Having said that, this is the first permission to require this kind of read/write access pattern, so I want to make sure we end up with something that future specs can also use. 

@jyasskin, would appreciate your input here...

Reply:
Agreed, and `{ access: "full" }` does seem to be a bit more future-proof if we ever need to make changes.

Also, see https://github.com/w3c/clipboard-apis/issues/51 for additional context about why we don't think separate "read" and "write" works well. For some reason, I thought you were on that thread.